addon/components/cfb-form-editor/question/options.js

```diff
@@ -83,6 +100,51 @@ export default RenderComponent.extend({
     this.setDirty();
   },
 
+  setupUIkit() {
+    UIkit.util.on("#option-list", "moved", (...args) =>
+      run(this, this._handleMoved, ...args)
+    );
+  },
```

Comment:
I know it's very convenient to solve this that way since you can just copy paste from the reorder question functionality. However the modern / new way to solve this would be a custom modifier. You should ask the person that gave you the assignment which way they'd prefer.

Reply:
To rewrite this using modifieres and still have it looking similar or as good as uikit would take a good amount of time, so this could just be an future issue of migrating to modifiers

Reply:
Huh? I meant a modifier that uses UIkit.

Reply:
E.g:

```hbs
<ul
  class="uk-list uk-list-divider uk-margin-remove-bottom uk-margin-small-top"
  id="option-list"
  {{uk-sortable @handle=".uk-sortable-handle" @onMoved=this._handleMoved}}
>
```

Reply:
It's just that we write code that has to be refactored to 100% since this thing here is not possible with glimmer components. I don't like throwaway code.

Reply:
I thought you meant to use https://github.com/ember-modifier/ember-modifier

Changed to use the uk-sortable component from ember-uikit

Reply:
Well I did mean that. My example is exactly that.

Reply:
However the sortable component is ok as well..